tests/coverage/test_gcovr.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -eu
+# for info see 'read.me'
+
+# return the canonicalized absolute pathname
+realpath() {
+  OURPWD=$PWD
+  cd "$(dirname "$1")"
+  LINK=$(readlink "$(basename "$1")")
+  while [ "$LINK" ]; do
+    cd "$(dirname "$LINK")"
+    LINK=$(readlink "$(basename "$1")")
+  done
+  REALPATH="$PWD/$(basename "$1")"
+  cd "$OURPWD"
+  echo "$REALPATH"
+}
+
+DIR_SRC=$(dirname "$(realpath $0)")
```

Comment:
SRC_DIR=$( cd "$(dirname "$0")" ; pwd -P )
BUILD_DIR=$PWD
then you deduce all other variables from these 2

Reply:
ok